dockstore-webservice/src/main/resources/migrations.1.14.0.xml

```diff
@@ -402,4 +402,43 @@
             <column name="engineversions" type="varchar"></column>
         </addColumn>
     </changeSet>
+    <changeSet author="ktran (generated)" id="addValidationMetrics">
+        <createTable tableName="validator_tool_validation_info">
+            <column name="validationstatusid" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="validator_tool_validation_info_pkey"/>
+            </column>
+            <column defaultValueComputed="now()" name="dbcreatedate" type="TIMESTAMP WITHOUT TIME ZONE"/>
+            <column name="dbupdatedate" type="TIMESTAMP WITHOUT TIME ZONE"/>
+            <column name="mostrecentisvalid" type="bool">
+                <constraints nullable="false"/>
+            </column>
+            <column name="mostrecentversion" type="varchar(255 BYTE)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="numberofruns" type="INTEGER">
+                <constraints nullable="false"/>
+            </column>
+            <column name="passingrate" type="float8">
+                <constraints nullable="false"/>
+            </column>
+            <column name="failedvalidationversions" type="varchar(255 BYTE)"/>
```

Comment:
Kinda wonder if these two columns should just be text https://www.postgresql.org/docs/current/datatype-character.html 
There's no performance difference anyway